ArchiveScraping/targetSpeaker.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+
+
+def findSpeaker():
+    try:
+        df = pd.read_csv('bloomberg.tv.2.csv')
+    except:
+        print("file could not be found in directory")
+
+
+    # apply function below to remove any words matching company from description
+    speaker = df.apply(removeCompany, axis=1)
+
+    # Remove interview and undecessary symbols
+    removalWords = ['Bloomberg TV Interview','BTV Interview','LIVE <GO> Interview',':','-']
+    for x in removalWords:
+        speaker = speaker.str.replace(x,'')
+
+    # remove things of the nature (113223 MM)
+    df['Speaker'] = speaker.str.replace(r'\([a-z,A-Z,0-9, ]*\)', '')
+
+
+    #output file to csv
+    df.to_csv('bloombergAltered.tv.2.csv')
+
+
```

Comment:
Not related to my selected text, but at least here we can reply to comments.
Yeah you're right. Theres several edge cases and inconsistent formatting. But we do need pieces of text that can be directly searchable in the snippets.
Can you try separating the title into a new field and add a .strip() to remove excess whitespace?

Reply:
I think once we have that, it should be searchable for at least 80% of rows.

Reply:
I added some lines and I think we have probably 90-95 percent of rows directly searchable now for the key speaker